### PR TITLE
Fixing print issue.

### PR DESCRIPTION
**Pull request by awais786**
ECOM-3781
Fixing print preview back-ground image issue.

### CODE REVIEW

**Comment by awais786** [comment]
@marcotuts Please check this fix.

**Comment by marcotuts** [comment]
I don't believe this addresses the root problem. Is there a second"-darker" image uploaded to s3 in the correct location? 

This proposed fix uses only the image that has been working 

That should be the fix instead of additional styling changes. The existing styles should handle watermark placement as expected, including rtl and responsive use cases.

**Comment by awais786** [comment]
@marcotuts 2nd image is also available.
[darker-image](https://github.com/edx/credentials/blob/master/credentials/static/images/xseries-seal-grayscale-darker.png)

And I have mentioned in the ticket image appear on print-preview also. But user need to click the print button and then cancel button twice. On third attempt image appears.

**Comment by marcotuts** [comment]
Discussed offline, reminded of intermittent nature of the bug. Thumbs up with the current approach, jus need a biapp wrapper around that left rule.

**Comment by marcotuts** [comment]
:thumbs:

File: credentials/static/sass/_views.scss [hunk 1]
```diff
@@ -92,6 +92,14 @@
         background-position: -325px -50px;
         background-size: 100% auto;
       }
+    .background-image{
+      position: absolute;
+      z-index: 0;
+      left: -325px;
```
Comment (marcotuts) [inline credentials/static/sass/_views.scss]: @include left(-325px);

**Comment by ahsan-ul-haq** [comment]
:+1:

**Pull request merged**